holidays/countries/gambia.py

```
@@ -0,0 +1,90 @@
+import holidays
+from hijri_converter import Gregorian
+
+def get_gambia_holidays(year):
+    gambia_holidays = {}
+    try:
+        gm_holidays = holidays.GM(years=year)
+        gambia_holidays.update(gm_holidays)
+    except KeyError:
+        gambia_holidays.update(_get_manual_gambia_holidays(year))
+    except Exception as e:
+        print(f"Error retrieving holidays for {year}: {e}")
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)